src/screens/Routes.js

```diff
@@ -19,6 +19,7 @@ import SendSheetWithData from './SendSheetWithData';
 import SettingsModal from './SettingsModal';
 import TransactionConfirmationScreenWithData from './TransactionConfirmationScreenWithData';
 import WalletScreen from './WalletScreen';
+import ViewPagerAdapter from 'react-native-tab-view-viewpager-adapter';
```

Comment:
- Can we just move this to after 'react'
- Once you've finalized, can you tidy up commit history

Thanks!

Reply:
@jinchung I fixed the import placement but we should wait till gesture recognition inside the pager is improved to merge